pkg/gui/presentation/commits.go

```diff
@@ -441,8 +441,15 @@ func displayCommit(
 	}
 
 	authorFunc := authors.ShortAuthor
-	if fullDescription {
+	switch common.UserConfig.Gui.CommitAuthorFormat {
+	case "short":
+		authorFunc = authors.ShortAuthor
+	case "full":
 		authorFunc = authors.LongAuthor
+	default:
+		if fullDescription {
+			authorFunc = authors.LongAuthor
+		}
```

Comment:
One last comment, though. For consistency's sake, let's use the `user_config_validation.go` file to validate the `commitAuthorFormat` setting instead of falling back to the `auto` setting.

Reply:
I think just the validation is good enough. I don't see a good enough reason to remove the default case.

I like the idea of pulling the values required for validation from the jsonschema or the struct tags, but I agree it is out of the scope of this PR. You can open up an issue and start a discussion about this if you're interested.